getting_started.Rmd

```diff
@@ -10,62 +10,71 @@ output:
 
-If you do not have a method of working with Git already it is recommended you use a Git client to help you manage your branches more easily. Install [GitKraken](https://www.gitkraken.com/) for a handy way to manage your course locally.
+If you do not have a method of working with Git already, we recommend using a Git client o facilitate easier branch management. Install [GitKraken](https://www.gitkraken.com/) for a convenient way to handle your course locally. 
 
 ***
 
 ## Starting a new OTTR course
 
 ### 1. Create a repository from this template
 
-In the upper right of the [landing page for this repository](https://github.com/jhudsl/OTTR_Template/tree/1.8), click the green `Use this template` button and follow the steps to set up your course's GitHub repository.
+On the [landing page of this repository](https://github.com/jhudsl/OTTR_Template/tree/1.8), locate the green "Use this template" in the upper right corner. Click on it and follow the provided instructions to configure your course's GitHub repository.
 
 \*If you want to use OTTR Website or a different spin off go to that respective repository:  
 
 - [OTTR Website](https://github.com/jhudsl/OTTR_Template_Website)
 
-![Where is the `Use this template` button?](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/template_button.png)
+![](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/template_button.png)
+
 
 ### 2. Name your repository and fill in a short description
 
-![Creating new course repository](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/creating_new_course_repo.png)
+Enter a name for your repository in the "Repository name" field and provide a brief description for it in the "Description" box.
+
+
+![](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/creating_new_course_repo.png)
+
 
-### 3.  Make sure your course is set to `public` for all GitHub actions to work
+### 3. Ensure your course is set to `public` for all GitHub Actions to work
+
+TODO: Clarify
 
 In particular, the rendered preview on pull requests won't work, but you can alternatively preview the content by re-rendering locally if you want to keep your course private. We will cover this later.
 
-### 4. Click the `Create repository from template button`
 
-Now you are ready to create your repository. Upon creating your repository, issues for getting your course started will be automatically filed in your repository. Go to `Issues` and you can follow them along to get set up.
+### 4. Select the `Create repository from template` button to proceed
+
+After creating your repository, you will find that issues related to starting your course have been automatically generated and filed in your repository. To begin the setup process, navigate to the Issues tab and follow the instructions provided in those issues.
+
+TODO: Issues are not generated automatically
```

Comment:
```suggestion
TODO: Issues are not generated automatically
```

Oh can you tell me where you saw that the issues didn't get made automatically? do you have a link to where this is? I'd like to look into it.

Reply:
I created [this repo](https://github.com/howardbaek/OTTR-Course) and the issues didn't get made automatically:

https://github.com/howardbaek/OTTR-Course/issues

Reply:
This happening all depends on permissions (again) which can be so varied depending on the repo type and person making the course. 

https://github.com/jhudsl/OTTR-Course/actions/runs/5480545242

We should see if there’s a way to make permissions more consistent or make it more clear that some of these things may not happen. I’ve been always wishing we could develop a better strategy for this. See https://github.com/jhudsl/OTTR_Template/issues/642

Reply:
That's a great idea.